tests/kernel/wave/wave_e2e_test.py

```diff
@@ -7,17 +7,29 @@
 import sympy
 import os
 import torch
+import json
 
 _run_e2e = int(os.environ.get("WAVE_RUN_E2E_TESTS", 0))
-
 require_e2e = pytest.mark.skipif(not _run_e2e, reason="e2e tests are disabled")
+default_test_shapes = [(1, 128), (256, 64), (256, 128), (256, 256), (256, 1024)]
+
+user_specified_test_shapes = ""
+
+test_params_path = os.environ.get("TEST_PARAMS_PATH", "")
+
+if os.path.isfile(test_params_path):

```

Comment:
Instead of checking `os.path.isfile(test_params_path)` we can do `os.environ.get("TEST_PARAMS_PATH", None)` instead and then test for `None`

Reply:
You mean something like this?
```
test_params_path = os.environ.get("TEST_PARAMS_PATH", None)

if test_params_path:
    with open(test_params_path, "r") as file:
        user_specified_test_shapes = json.load(file)
```

Reply:
yes